test/index.js

```diff
@@ -0,0 +1,61 @@
+const seleniumAssistant = require('selenium-assistant');
+const serve = require('serve');
+const path = require('path');
+const Mocha = require('mocha');
+const chai = require('chai');
+
+async function initTest() {
+  console.log('downloading browsers...');
+  const expiration = 24;
+  await seleniumAssistant.downloadLocalBrowser('chrome', 'stable', expiration);
+
+  console.log('starting server...');
+  const serveDir = path.join(__dirname, '../');
+  server = serve(serveDir, {
+      port: 6881,
+  });
+
+  const browsers = seleniumAssistant.getLocalBrowsers();
+  browsers.forEach((browser) => {
```

Comment:
Nit: `browsers.forEach(browser => `

A singular argument doesn't require enclosure.

Reply:
i'll let prettier take care of this